terraform/modules/database/main.tf

```diff
@@ -0,0 +1,23 @@
+resource "aws_db_instance" "api3-operations" {
+  identifier             = "api3-operations"
+  instance_class         = "db.t3.micro"
+  allocated_storage      = 5
+  engine                 = "postgres"
+  engine_version         = "13.1"
+  username               = var.db_username
+  password               = var.db_password
+  db_subnet_group_name   = aws_db_subnet_group.api3-operations.name
+  vpc_security_group_ids = [aws_security_group.rds.id]
```

Comment:
I think there should be 'rds' module to be defined like [here](https://github.com/hashicorp/learn-terraform-rds/blob/main/main.tf#L28). 
In my opinion, we don't need to expose our ports publicly as in the link _^.